t_reconf/test_reconf_base.py

```diff
@@ -1068,13 +1068,6 @@ def test_reconf_server_queue_size(
             expect_502_statuses,
         )
 
-        self.assertTrue(
```

Comment:
Please also remove `dmesg_cond` param. And, probably, we should add a check for perfstat here?